### DISCUSSION TITLE
Permalink as option in Reports

### DISCUSSION THREAD

**Discussion by SeanStock-THG** [apache/superset#22905] [Ideas]
Something like this would be a great addition allowing users to send out reports with custom filtering applied to the dashboards. 

![image](https://user-images.githubusercontent.com/108939297/215457205-aa38add8-c00c-4104-88d8-33be32213262.png)

**Comment by Pedrowach** [comment]
Any updates?

**Reply by SeanStock-THG** [reply]
I believe the plan is to integrate dashboard filters that will ultimately give the same result.
https://preset.io/blog/apache-superset-4-0-release-notes/#alerts-and-reports-modal-redesign
